react/Icon/index.jsx

```diff
@@ -1,8 +1,10 @@
-import React from 'react'
+import React, { PropTypes as Types } from 'react'
```

Comment:
https://facebook.github.io/react/docs/typechecking-with-proptypes.html
> React.PropTypes has moved into a different package since React v15.5. Please use the prop-types library instead.

Unless it's added with `preact-compat`. But I think to be more reliable, we should depend on the real [proptypes module](https://github.com/facebook/prop-types).

On the other hand, I cannot understand why you alias the import.

Reply:
> On the other hand, I cannot understand why you alias the import.

Oh I'm sure you can :D : it is to type les characters. It is fairly common : https://github.com/search?utf8=%E2%9C%93&q=%22PropTypes+as+Types%22&type=Code

Reply:
What do you mean by to be more reliable ?

Reply:
`preact-compat` is using the `prop-types` module under the hood 👌 https://github.com/developit/preact-compat/blob/master/package.json#L87

Reply:
We are using `preact` but to stay more reliable on `react`, we also use `preact-compat`.
For instance `preact` let us declare the `render` function with `props` as parameters, `react` doesn't. So we usen't.

The same goes to `PropTypes`, since `react` 15.5, `PropTypes` has moved to a different package.

If you look at the `import` part of both below codesandbox, you'll understand:

- preact with preact-compat works with `import { PropTypes } from "preact";` (https://codesandbox.io/s/2okk0663rp)
- react don't (https://codesandbox.io/s/kmj356yp15)

On the other hand, both works with `import PropTypes from "prop-types"`.

In any event, both used the exact same release of `prop-types`.

Reply:
done